tests/python/pants_test/engine/test_isolated_process.py

```diff
@@ -198,46 +375,40 @@ def test_javac_compilation_example(self):
     self.assertTrue(os.path.exists(os.path.join(classpath_entry.path, 'simple', 'Simple.class')))
 
   def test_javac_compilation_example_rust_success(self):
```

Comment:
Drop rust from the name of this test

Reply:
Done, and also removed from the other test with `_rust_` in the name.